tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetMvc5IastTests.cs

```diff
@@ -70,6 +71,7 @@ public AspNetMvc5IastTests(IisFixture iisFixture, ITestOutputHelper output, bool
         SetEnvironmentVariable("DD_IAST_REQUEST_SAMPLING", "100");
         SetEnvironmentVariable("DD_IAST_MAX_CONCURRENT_REQUESTS", "100");
         SetEnvironmentVariable("DD_IAST_VULNERABILITIES_PER_REQUEST", "100");
+        SetEnvironmentVariable("DD_TRACE_DEBUG", "1");
```

Comment:
did you want to commit that?  it slows down tests quite a lot

Reply:
You are right. It was not intended... Thanks!